lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
Well... technically, yes. But it occured to me that if I was a Kuzzle administrator, I may also want to limit the number of fetched documents using `mGet`, for performances and bandwidth reasons. And I thought the best way to do that was to apply `maxMultiActionsCount` to this API route too.

What do you think?